projects/1.20/assets/pams-harvestcraft-2-food-extended/pamhc2foodextended/lang/zh_cn.json

```
@@ -0,0 +1,874 @@
+  "item.pamhc2foodextended.maplesyrupwafflesitem": "枫糖华夫饼",
+  "item.pamhc2foodextended.marzipanitem": "杏仁蛋白软糖",
+  "item.pamhc2foodextended.mashedpotatoeschickenbiscuititem": "马铃薯泥佐鸡肉烤软饼",
+  "item.pamhc2foodextended.mashedsweetpotatoesitem": "红薯泥",
+  "item.pamhc2foodextended.mcpamitem": "酸菜牛排堡",
+  "item.pamhc2foodextended.meatfeastpizzaitem": "肉宴匹萨",
+  "item.pamhc2foodextended.meatloafsandwichitem": "肉饼三明治",
+  "item.pamhc2foodextended.meatpieitem": "肉馅饼",
+  "item.pamhc2foodextended.meatystirfryitem": "肉炒",
+  "item.pamhc2foodextended.meesuaitem": "潮汕咸面线",
+  "item.pamhc2foodextended.melonjellysandwichitem": "西瓜酱三明治",
+  "item.pamhc2foodextended.meringuebombeitem": "水果甜点",
+  "item.pamhc2foodextended.meringuebrownieitem": "蛋白霜布朗尼",
+  "item.pamhc2foodextended.meringuecookieitem": "蛋白霜曲奇",
+  "item.pamhc2foodextended.meringueitem": "蛋白霜",
+  "item.pamhc2foodextended.meringuerouladeitem": "蛋白霜瑞士卷",
+  "item.pamhc2foodextended.merveilleuxitem": "比利时惊奇巧克力蛋糕",
+  "item.pamhc2foodextended.mettbrotchenitem": "肉末洋葱面包",
+  "item.pamhc2foodextended.mincepieitem": "水果馅饼",
+  "item.pamhc2foodextended.mintchocolatechipicecreamitem": "薄荷巧克力冰淇淋",
+  "item.pamhc2foodextended.misopasteitem": "味噌",
+  "item.pamhc2foodextended.misosoupitem": "味噌汤",
+  "item.pamhc2foodextended.mixedflowerssaladitem": "繁花沙拉",
+  "item.pamhc2foodextended.mixedsaladitem": "什锦沙拉",
+  "item.pamhc2foodextended.mochaicecreamitem": "摩卡冰激凌",
+  "item.pamhc2foodextended.mochicakeitem": "麻薯蛋糕",
+  "item.pamhc2foodextended.mochidessertitem": "麻薯",
+  "item.pamhc2foodextended.mochiitem": "麻薯",
+  "item.pamhc2foodextended.molasseschickenitem": "蜜汁鸡肉",
+  "item.pamhc2foodextended.mozzerellasticksitem": "奶酪棒",
+  "item.pamhc2foodextended.mulberryjellyitem": "桑葚酱",
+  "item.pamhc2foodextended.mulberryjellysandwichitem": "桑葚酱三明治",
+  "item.pamhc2foodextended.mulberryjellytoastitem": "桑葚酱吐司",
+  "item.pamhc2foodextended.mulberryjuiceitem": "桑葚汁",
+  "item.pamhc2foodextended.mulberrypieitem": "桑葚派",
+  "item.pamhc2foodextended.mulberrysmoothieitem": "桑葚沙冰",
+  "item.pamhc2foodextended.mulberryyogurtitem": "桑葚酸奶",
+  "item.pamhc2foodextended.museliitem": "杂锦果麦",
+  "item.pamhc2foodextended.mushroomketchupitem": "蘑菇沙司",
+  "item.pamhc2foodextended.mushroomketchupomeletitem": "蘑菇酱煎蛋饼",
+  "item.pamhc2foodextended.mushroomlasagnaitem": "蘑菇千层面",
+  "item.pamhc2foodextended.mushroomoilitem": "蘑菇油",
+  "item.pamhc2foodextended.mushroomrisottoitem": "牛肝菌烩饭",
+  "item.pamhc2foodextended.mushroomsteakitem": "蘑菇牛排",
+  "item.pamhc2foodextended.mustarditem": "芥末",
+  "item.pamhc2foodextended.musubiitem": "夏威夷火腿饭团",
+  "item.pamhc2foodextended.naanitem": "印度烤饼",
+  "item.pamhc2foodextended.nachoesitem": "干酪玉米片",
+  "item.pamhc2foodextended.neapolitanicecreamitem": "那不勒斯冰淇淋",
+  "item.pamhc2foodextended.nopalessaladitem": "诺帕莱斯沙拉",
+  "item.pamhc2foodextended.nuttoppedicecreamitem": "“尖”果冰淇淋",
+  "item.pamhc2foodextended.oatmealraisincookiesitem": "燕麦葡萄干饼干",
+  "item.pamhc2foodextended.okrachipsitem": "秋葵心片",
```

```suggestion
  "item.pamhc2foodextended.okrachipsitem": "秋葵片",
```
秋葵的芯是软的空的，类似尖椒，这个只能算横切片吧